__tests__/groundplaces/updateStopCluster.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { GroundPlaces } from '../../src/classes/groundplaces';
+import { fakeGroundPlacesJSON } from '../../mocks/groundPlaces';
+
+describe('updateStopCluster()', () => {
```

Comment:
#updateStopCluster instead of updateStopCluster()

It's our convention.

Reply:
Ok done for all tests scenarios